draft-ietf-core-object-security.md

```diff
@@ -945,30 +949,72 @@ Note that the HTTP Status Code 200 in the next-to-last message is the mapping of
 
 ## CoAP-to-HTTP Translation Proxy 
 
+Section 10.1 of {{RFC7252}} describes the behavior of a CoAP-to-HTTP proxy.  RFC 8075 {{RFC8075}} does not cover this direction in any more detail and so an example instantiation of Section 10.1 of {{RFC7252}} is used below.
+
+Example:
+
+~~~~~~~~~~~
+[CoAP request -- Before object security processing]
+
+  GET /hc/http://device.local/orders
+  Uri-Host: proxy.local
+
+[CoAP request -- CoAP Client to Proxy]
 
+  POST /
+  Uri-Host: proxy.local
+  Object-Security: [empty]
```

Comment:
You should add a Proxy-Uri option here.  I am not sure how it would be parsed from the original query as it does not seem to match the rules that were provided earlier in the document about moving part of the proxy-uri into the encryption.

Reply:
POST /
Uri-Host: proxy.local
Proxy-Uri: http://device.local
Object-Security: [empty]
Payload: .... [binary]